websites/docs-smoke-test/src/content/components/cards.mdx

```diff
@@ -328,14 +328,44 @@ Icons on **narrow** cards are aligned to the left of the card regardless of whet
 
 ```jsx
 <Cards>
-  <Card image={<CardBannerOSSIcon/>}>
+  <Card image={<CardBannerApiIcon />}>
+    Flat card with an image
+  </Card>
+  <Card image={<CardBannerDefaultIcon />}>
+    Flat card with an image
+  </Card>
+  <Card image={<CardBannerMcIcon />}>
+    Flat card with an image
+  </Card>
+  <Card image={<CardBannerOptional1Icon />}>
+    Flat card with an image
+  </Card>
+  <Card image={<CardBannerOptional2Icon />}>
+    Flat card with an image
+  </Card>
+  <Card image={<CardBannerOssIcon />}>
```

Comment:
This results in this:

<img width="770" alt="image" src="https://user-images.githubusercontent.com/1110551/162401067-f00ec602-4e8a-46fd-b898-53356e4f5115.png">